tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/UpToDate/BuildUpToDateCheckTests.cs

```diff
@@ -1419,17 +1419,16 @@ public async Task IsUpToDateAsync_False_Kinds_InputNewerThanOutput_NoKindIgnored
             await AssertNotUpToDateAsync(
                 $"""
                 Adding UpToDateCheckOutput outputs:
-                    C:\Dev\Solution\Project\TaggedOutput
                     C:\Dev\Solution\Project\Output
+                    C:\Dev\Solution\Project\TaggedOutput
                 Adding UpToDateCheckBuilt outputs:
-                    C:\Dev\Solution\Project\TaggedBuilt
                     C:\Dev\Solution\Project\Built
+                    C:\Dev\Solution\Project\TaggedBuilt
                 Adding project file inputs:
                     {_projectPath}
                 Adding newest import input:
                     {_projectPath}
                 Adding UpToDateCheckInput inputs:
-                    C:\Dev\Solution\Project\TaggedInput
                     C:\Dev\Solution\Project\Input
```

Comment:
An example of where the test expectation needed to be updated. The order here now matches that earlier within this test method where they're added to the snapshot.